src/test_typing_extensions.py

```diff
@@ -5066,6 +5066,47 @@ def test_cannot_combine_closed_and_extra_items(self):
             class TD(TypedDict, closed=True, extra_items=range):
                 x: str
 
+    def test_typed_dict_signature(self):
+        self.assertListEqual(
+            list(inspect.signature(TypedDict).parameters),
+            ['typename', 'fields', 'total', 'closed', 'extra_items', 'kwargs']
+        )
+
+    def test_inline_too_many_arguments(self):
+        with self.assertRaises(TypeError):
+            TypedDict[{"a": int}, "extra"]
+
+    def test_inline_not_a_dict(self):
+        with self.assertRaises(TypeError):
+            TypedDict["not_a_dict"]
```

Comment:
Could also add a test for a single-element tuple, to make it clear that it's intentionally disallowed (at least until we have a use case)?

```suggestion
            TypedDict["not_a_dict"]

        # a tuple of elements isn't allowed, even if the first element is a dict:
        with self.assertRaises(TypeError):
            TypedDict[({"key": int},)]
```